mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -1,22 +1,38 @@
-
-import {Navigate, useParams} from "react-router-dom";
+import {Navigate, Params, useParams} from "react-router-dom";
+import {Button} from "src/component/button/Button";
 import {HeadingLevel, Title} from "src/component/title/Title";
+import {WayPreviewDAL} from "src/dataAccessLogic/WayPreviewDAL";
 import {OwnWaysTable} from "src/logic/waysTable/OwnWaysTable";
 import {pages} from "src/router/pages";
 import styles from "src/logic/userPage/UserPage.module.scss";
 
+/**
+ * Query param types
+ */
+interface QueryParamTypes extends Params {
+
+  /**
+   * User's uuid
+   */
+  uuid: string;
+}
+
 /**
  * User page
  */
 export const UserPage = () => {
-  const {uuid} = useParams();
+  const {uuid} = useParams() as QueryParamTypes;
```

Comment:
What will happened if uuid is undefined (handle this scenario)

Reply:
If uuid is undefined  client will be navigate to page 404. This scenario works by code on  30 line. 
But I dont know is it right to make interface for uuid from useParams as string?